models/email_templates/invoice_email.yml

```diff
@@ -205,13 +205,15 @@ properties:
           formatted:
             type: string
           value:
-            type: integer
+            type: float
       meta:
         type: object
         properties:
           mandate_url:
             type: string
             description: Link to the confirmation page in the Stripe.
+          mandate_tag:

```

Comment:
Should we put add a description here? It seems like this relates to Stripe direct debit transactions (https://stripe.com/docs/sources/sepa-debit) but I didn't see anything in their docs about tags. Will readers understand what this is?

Reply:
@markcmurphy I received the following description for `mandate_tag`. 
`mandate_tag`: Short name of the payment document
See the DEVDOCS ticket for more information.